src/main/java/org/stellar/sdk/Transaction.java

```diff
@@ -195,17 +198,81 @@ public int getFee() {
    */
   public String toEnvelopeXdrBase64() {
     try {
-      org.stellar.sdk.xdr.TransactionEnvelope envelope = this.toEnvelopeXdr();
+      TransactionEnvelope envelope = this.toEnvelopeXdr();
       ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
       XdrDataOutputStream xdrOutputStream = new XdrDataOutputStream(outputStream);
-      org.stellar.sdk.xdr.TransactionEnvelope.encode(xdrOutputStream, envelope);
+      TransactionEnvelope.encode(xdrOutputStream, envelope);
       Base64 base64Codec = new Base64();
       return base64Codec.encodeAsString(outputStream.toByteArray());
     } catch (IOException e) {
-      throw new AssertionError(e);
+      throw new RuntimeException(e);
     }
   }
 
+  /**
+   * Decodes a base64-encoded XDR {@link TransactionEnvelope}.
+   *
+   * @bug Does not work on Android, as it performs a call to the method
+   *      decode() of class org.apache.commons.codec.binary.Base64, which
+   *      conflicts with a legacy Apache class included in Android.
+   *      To decode a base64-encoded {@link TransactionEnvelope} on Android,
+   *      use {@code android.util.Base64.decode()}, then pass the result to
+   *      the method {@link #decodeXdrEnvelope(byte[])}.
```

Comment:
Please use `com.google.common.io.BaseEncoding`.

Reply:
Will do.

Reply:
I am also removing the whole `@bug` comment, which is now obsolete.